package.json

```diff
@@ -21,7 +21,7 @@
     "optimize:png": "find source/images -type f -name '*.png' -print0 | xargs -0 -n 1 -P 6 optipng",
     "optimize": "run-p optimize:**",
     "server": "pipenv run python network-api/manage.py runserver",
-    "start": "npm i && npm run build-uncompressed && run-p server watch:**",
+    "start": "npm i && run-p build-uncompressed server watch:**",
```

Comment:
mostly a dev Quality of Life change - this starts the server _way_ earlier, so you can just reload the tab once webpack finishes the build, instead of getting a "server not found" for like... a minute